testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -1518,3 +1518,119 @@ def test_replication_attachments_survive_channel_removal(setup_client_syncgatewa
         with open('resources/data/sample_text.txt', 'r') as file:
             expected_text = file.read()
         assert expected_text == att
+
+
+@pytest.mark.sanity
+@pytest.mark.listener
+@pytest.mark.syncgateway
+@pytest.mark.replication
+@pytest.mark.session
+def test_verify_doc_updates_pull(setup_client_syncgateway_test):
+    """
+    1. Add docs with attachments
+    2. Start push/pull replication
```

Comment:
can you mention here start push/pull replication with continuous true

Reply:
Done